### DISCUSSION TITLE
Stop verdaccio setting default registry on startup

### DISCUSSION THREAD

**Discussion by atomicrobokid** [verdaccio/verdaccio#4886] [Q&A] [answered]
Hi,

I may have overlooked some config, but i'm trying to run Verdaccio as a local development tool only, and purely for some scoped packages, however when i start the server i see:

```
Set npm registry to http://localhost:4873/
Set npm @mycompany:registry to http://localhost:4873/
Set yarn registry to http://localhost:4873/
Set yarn @mycompany:registry to http://localhost:4873/
```

Which means when i stop the server, i need to rememeber to revert the changes in the global npmrc file, so isn't ideal.

I'd like it to _just_ set the scoped package registry config, not the default registry, if possible

**Comment by mbtools** [comment, marked as answer]
AFAIK, Verdaccio does not make any such changes to your local settings. What command do you use to start it?

**Reply by juanpicado** [reply]
What I don't understand what the role of verdaccio on this question because looks like a package manager question 🤔

**Reply by atomicrobokid** [reply]
Aha, so this is using NX's implementation of Verdaccio. Thanks both for your answers as this helps point me in their direction. 

They don't expose much config so assumed it was just a really thin layer on top of Verdaccio which was setting those values. I'll raise with them 👍
